content/learn/getting-started/node-javascript-language.fr.md

```diff
@@ -0,0 +1,40 @@
+---
+title: how-much-javascript-do-you-need-to-know-to-use-nodejs
+displayTitle: 'Combien de JavaScript devez-vous connaître pour utiliser Node.js ?'
+description: 'Si vous débutez avec JavaScript, à quel point devez-vous connaître le langage ?'
+authors: flaviocopes, ollelauribostrom, MylesBorins, fhemberger, LaRuaNa, ahmadawais, elanandkumar, AugustinMauroy
+category: learn
+---
+
+En tant que débutant, il est difficile d'arriver à un point où vous êtes suffisamment confiant dans vos capacités de programmation.
+
+Lorsque vous apprenez à coder, vous pouvez aussi vous demander où s'arrête JavaScript et où commence Node.js, et vice versa.
+
+**Je vous recommande d'avoir une bonne maîtrise des principaux concepts JavaScript avant de plonger dans Node.js:**.
+
+* Lexical Structure
+* Expressions
+* Types
+* Classes
+* Variables
+* Functions
+* this
+* Arrow Functions
+* Loops
+* Scopes
+* Arrays
+* Template Literals
+* Semicolons
+* Strict Mode
+* ECMAScript 2015 and beyond
```

Comment:
Il n'y a pas de traduction ici (pas très grave ceci dit, je voulais juste le faire remarquer au cas où c'était involontaire)

Reply:
Normale car se sont des nom javascript.
Après si tu pense qu'il faut les traduire on peu

Reply:
Je crois que ce sont les noms des autres pages (e.g. tu as traduit le titre de la page `ECMAScript 2015 and beyond` par `ECMAScript 2015 (ES6) et plus`)

Reply:
Oui je vais ça demain